tools/make_nr/make_nr.xml

```diff
@@ -51,6 +51,14 @@ python $__tool_directory__/make_nr.py $alphasort -s '$separator' -o '$output'
             <param name="alphasort" value="-a"/>
             <output name="output" file="deduplicate.sortids.fasta" ftype="fasta"/>
         </test>
+        <test>
+            <param name="input" value="empty.fasta" ftype="fasta"/>
+            <output name="output" file="empty.fasta" ftype="fasta"/>
+        </test>
+        <test>
+            <param name="input" value="empty.fasta,empty.fasta" ftype="fasta"/>
+            <output name="output" file="empty.fasta" ftype="fasta"/>
+        </test>
```

Comment:
That's elegant ❤️

But we'll need to add ``empty.fasta`` to the tool manifest in ``tools/make_nr/.shed.yml``

Reply:
will do. lets see if this elegant solution is also a working solution :)